daemon/src/maker_inc_connections.rs

```diff
@@ -607,7 +607,7 @@ async fn upgrade(
 
     let daemon_version = daemon_version.unwrap_or_else(|| String::from("<= 0.4.7"));
 
-    tracing::trace!(%taker_id, %taker_address, %negotiated_wire_version, %daemon_version, "Connection upgrade successful");
+    tracing::debug!(%taker_id, %taker_address, %negotiated_wire_version, %daemon_version, "Connection upgrade successful");
```

Comment:
🤔 This was purposefully demoted recently because it is quite low level compared to the "Connection is ready" log which is emitted once the connection has actually been established. It's not a big deal, but I obviously disagree with this change 😛

Reply:
Fair enough :) 
In a recent debugging session we identified the need for more logs, but it might be easier to just enable this using `RUST_LOG`. I'll drop this commit from the PR.

Reply:
Perhaps we should just drop the `Connection is ready` log instead?

Reply:
I'll leave it for now.